packages/java-edition/src/mcfunction/parser/argument.ts

```diff
@@ -445,10 +444,10 @@ function entity(
 				children: [res],
 			}
 
-			if (core.StringNode.is(res)) {
-				ans.playerName = res
-			} else if (EntitySelectorNode.is(res)) {
+			if (EntitySelectorNode.is(res)) {
 				ans.selector = res
+			} else if (core.StringNode.is(res)) {
+				ans.playerName = res
```

Comment:
I can't immediately see why this would break, but what's the reason for switching these around?

Reply:
It was just to avoid any @ selectors getting picked up as potential player names (or UUIDs as they had been in the past)